frontend/www/js/omegaup/common/index.ts

```diff
@@ -2,9 +2,27 @@ import Homepage from '../components/homepage/Homepage.vue';
 import { OmegaUp } from '../omegaup';
 import { types } from '../api_types';
 import Vue from 'vue';
+import * as ui from '../ui';
+import T from '../lang';
 
 OmegaUp.on('ready', () => {
   const payload = types.payloadParsers.IndexPayload();
+  const commonPayload = types.payloadParsers.CommonPayload();
+  console.log(commonPayload.pendingDaysForVerification);
+
+  if (commonPayload.pendingDaysForVerification < 5) {
+    ui.warning(
+      T.accountVerifyWarning +
+        commonPayload.pendingDaysForVerification +
+        T.days,
+    );
+  } else if (commonPayload.pendingDaysForVerification <= 7) {
+    ui.warning(
+      T.accountVerifyWarning +
+        commonPayload.pendingDaysForVerification +
+        T.days,
+    );
```

Comment:
Here you can use string interpolation as well:
```suggestion
      ui.warning(
        ui.formatString(T.accountVerifyWarning, {
          days: commonPayload.pendingDaysForVerification,
        }),
      );
```